apps/api/src/app/shared/dtos/message-template.ts

```diff
@@ -30,6 +30,10 @@ export class MessageTemplate {
   @IsString()
   feedId?: string;
 
+  @IsOptional()
+  @IsString()
+  layoutId?: string;
+
```

Comment:
```suggestion
  @IsOptional()
  layoutId?: string | null;
```
I am afraid that the validation might make this fail.